waspc/e2e-test/test-outputs/waspJob-golden/waspJob/.wasp/out/Dockerfile

```diff
@@ -2,7 +2,7 @@
 #   Because if not, we had situations where it would use the different version
 #   locally and on Github CI. This way we ensure exact version is used,
 #   and also have control over updating it (instead of update surprising us).
-FROM node:18-alpine3.17 AS node
+FROM node:20-alpine3.17 AS node
```

Comment:
Node version is also used here. So it will be set to the node version that we use as a lower boundary. If they use node 21 during development and we have node 20 here -> hm, how problematic is that? @infomiho @sodic .